src/components/EditorFooter/index.tsx

```diff
@@ -42,7 +42,9 @@ const EditorFooter = () => {
             mb="0"
             color="brand.primary"
             data-testid="ruler-label"
-            w="70px"
+            w="85px"
+            h="40px"
+            pt="3px"
```

Comment:
Please do not use abbreviations anymore. Change it to full name such as width, height or margin.

Reply:
got it, thanks